doc/nrf/device_guides/working_with_nrf/nrf70/developing/power_profiling.rst

```diff
@@ -0,0 +1,81 @@
+.. _ug_nrf70_developing_power_profiling:
+
+Power profiling
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Power Profiler Kit II (PPK2) of Nordic Semiconductor can be used to measure the power consumption of nRF70 Series devices in Low-power mode.
```

Comment:
Seems that the below guide is tailored to the nRF7002 DK (not generic enough for all kits)